docs/configuration.md

```diff
@@ -2953,6 +2953,22 @@ the end marker will be deleted.
 > - there are comments before the end marker, as without the end marker they
 >   would be treated as outside of the optional-braces region.
 
+### `rewrite.scala3.countEndMarkerLines`
+
+> Since v3.0.6.
+
+This flag dictates which part of the expression terminated by the end marker
+is used to calculate the span for the purposes of applying
+[`insertEndMarkerMinLines`](#rewritescala3insertendmarkerminlines) and
+[`removeEndMarkerMaxLines`](#rewritescala3removeendmarkermaxlines).
+
+- `all` (default): the entire expression
+- `lastBlockOnly`: only the last block with significant indentation relative to
```

Comment:
Wdyt about making `lastBlockOnly` a default instead of `all`?

I understand that it changes the previous behavior but in this case there aren't a lot of codebases that will be affected.

Reply:
why would we do that? I'm introducing this option just for you but it's not the most logical option.

Reply:
I might be wrong, but I have a strong feeling that this new one is more close to expected one from user perspective.
Anyway feel free to ignore this comment.

The only thing I'm worried is that if there will be more people like me it will save some time from questions/issues on this topic.